test/pdfs/.gitignore

```diff
@@ -139,6 +139,7 @@
 !issue3205r.pdf
 !issue3207r.pdf
 !issue3263r.pdf
+!bug1661226.pdf
```

Comment:
I already included a file with a pushbutton in https://github.com/mozilla/pdf.js/commit/0fe80df2a75a70162ab3bae56eb9fd943f4726ab#diff-b19c886c212474d3b37d6026a795a246. The reason it didn't cause rendering problems is because it's not also run with `forms: true`.

Instead of adding this new file, please use the existing one by reverting the changes in the `test` folder in this PR and including a new entry in the `test_manifest.json` file as follows:

```
    {  "id": "issue4872",
       "file": "pdfs/issue4872.pdf",
       "md5": "21c6cbc682140d6f6017bbeb45892053",
       "rounds": 1,
       "link": true,
       "firstPage": 1,
       "lastPage": 1,
       "type": "eq",
       "forms": true
    },
```

I have verified that the bug also happens for that file.